docs/tutorials-samples/targeting-core-for-new-libraries.md

```diff
@@ -0,0 +1,298 @@
+# Targeting with .NET Core Libraries
+
+Targeting .NET Core for new libraries can be done a number of ways, depending on the platform(s) you wish to support.
+
+## How do I target .NET Core?
+
+First things first:
+
+For the purposes of building a library, targeting ".NET Core" means targetting the .NET Standard Platform.  [This document](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md) outlines everything you need to know, in-depth, about what that means.
```

Comment:
OCD comment: splitting your lines in markdown makes the file more readable when no formatting is applied, without the risk of super-long lines.